src/components/timeline/maptimeline/LayerRowPeriodItem.js

```diff
@@ -0,0 +1,46 @@
+import {connect} from 'react-redux';
+import {setState} from '@jvitela/recompute';
+import {utils} from '@gisatcz/ptr-utils';
+
+import mapTimelineSelectors from '../../../state/Timeline/MapTimeline/selectors';
+import periodsSelectors from '../../../state/Periods/selectors';
+import periodsActions from '../../../state/Periods/actions';
+
+const componentId = `layer_row_period_item_${utils.uuid()}`;
+
+const mapStateToProps = (state, ownProps) => {
+	setState(state);
+
+	let standardized = null;
+	if (ownProps?.itemPeriod?.key) {
+		standardized = periodsSelectors.getByKey(state, ownProps.itemPeriod.key);
+	} else {
+		standardized = ownProps.itemPeriod;
+	}
+	const active = mapTimelineSelectors.getTimelineLayerPeriodActive(
+		ownProps.mapKey,
+		ownProps.layer,
+		ownProps.itemPeriod
+	);
+
+	return {
+		standardized,
```

Comment:
what is standardized?

Reply:
Renamed to the parsedPeriod 1152a2c1acdb5bc5991bfe3ebea8e5eb9f20a4ad